app/views/content/what-is-qts.md

```diff
@@ -19,12 +20,25 @@ keywords:
   - State-school
 ---
 
-Qualified teacher status (QTS) is what you need to teach in a state school in England. 
+Qualified teacher status (QTS) is what you need to teach in most primary and secondary schools in England.
 
-It is a confirmation that you have the right training and meet the standards required to be a teacher. 
+It is a confirmation that you have the right training and meet the standards required to be a teacher.
 
-All postgraduate teacher training courses in England lead to QTS. You can see the different types of courses available by [searching for postgraduate teacher training](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+## How to get QTS
+
+All postgraduate teacher training courses in England lead to QTS. You can see the different types of courses available by [searching for postgraduate teacher training]
+(https://www.gov.uk/find-postgraduate-teacher-training-courses).
+
+Entry requirements for QTS courses include having an undergraduate degree and GCSEs at level 4/grade C or above in Maths and English. You’ll also need a level 4/grade C or above
+in Science if you want to be a primary school teacher. As well as offering QTS, many courses also offer a PGCE or PGDE. Find out about [PGCE and PGDE](/what-is-a-pgce).
 
 QTS is a teaching standard that is respected across the world. Having QTS means you can move around the country and teach in any state school.
```

Comment:
Does this line ('QTS is a teaching standard....') work better as the last para in the intro?

But ignore this if you've already considered and placed here deliberately!